helm/charts/determined/values.yaml

```diff
@@ -377,3 +377,16 @@ genai:
 
     ## Size of the generated PVC
     storageSize: 100Gi
+
+  ## Extra Resource Pool Metadata is hardcoded information about the
+  ## GPUs available to the resource pools. This information
+  ## is not provided in k8s so we provide it directly.
+  ## Note: All resource pools defined here need to also be reflected in
+  ## the .Values.resourcePools.
+  extra_resource_pool_metadata:
+    a100:
+      gpu_type: A100
+      max_agents: 3
+    v100:
+      gpu_type: V100
+      max_agents: 2
```

Comment:
could we comment it out? because this default is not universal and needs a user override.

Reply:
Yup, done